modules/update-mco-process.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="mco-update-process_{context}"]
+= Machine Config Operator node updates
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and compute node. During this process, the MCO performs the following sequential actions in the cluster:
```

Comment:
another little nit: I ain't a grammar specialist in any way, but "During the process, MCO performs the following sequential actions" sounds a bit better than "During this process, **the** MCO performs **the** following sequential actions"   
same for "**The** CVO..." which here and there mid-sentence may be just "CVO" as well?

Reply:
ain't a native speaker however, so take that with a grain of salt ;)

Reply:
I think in these cases, "MCO" and "CVO" need an article before them such as "the", and looking around the rest of the docs, it seems that this is generally how these Operators are referred to. I think one prominent exception would be something like OSUS, where it would sound somewhat awkward to say "the OSUS does x".

Reply:
for consistency sake, and since its a definite bikeshedding from my side :) leaving this one up to you. however not in all but in some of the occasions of its use it does sound somewhat awkward, especially when a sentence is not talking about cvo itself, but about some other object, which being impacted by cvo somehow.